cpp/oneapi/dal/algo/svm/backend/model_interop.hpp

```diff
@@ -29,14 +32,27 @@ class model_interop : public base {
 };
 
 template <typename DaalModel>
-class model_interop_impl : public model_interop {
+class model_interop_impl : public model_interop,
+                           public ONEDAL_SERIALIZABLE(svm_model_interop_impl_multiclass_id) {
 public:
+    model_interop_impl() = default;
+
     model_interop_impl(DaalModel& model) : daal_model_(model) {}
 
     const DaalModel get_model() const {
         return daal_model_;
     }
 
+    void serialize(dal::detail::output_archive& ar) const override {
+        dal::backend::interop::daal_output_data_archive daal_ar(ar);
+        daal_ar.setSharedPtrObj(const_cast<DaalModel&>(daal_model_));
```

Comment:
Seems like `daal_model_` is not a `DaalModel` but a `SharedPtr<DaalModel>` or `DaalModelPtr` at least

Reply:
`DaalModel` means shared pointer to model, probably need to rename